cudax/include/cuda/experimental/__async/basic_sender.cuh

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2024 NVIDIA Corporation
+ *
+ * Licensed under the Apache License Version 2.0 with LLVM Exceptions
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   https://llvm.org/LICENSE.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cuda/std/__utility/move.h>
+
+#include "completion_signatures.cuh"
+#include "config.cuh"
+#include "cpos.cuh"
+#include "utility.cuh"
+
+// This must be the last #include
+#include "prologue.cuh"
```

Comment:
Please add this to the `.clang-format` file as a separate rule

Reply:
in the `.clang-format` file, why is `Priority` set differently from `SortPriority`? And why is there no header category at `Priority: 1`?